examples/boussinesq_2d_imex/playground.py

```diff
@@ -36,12 +36,12 @@
     swparams['do_LU'] = True
 
     sparams = {}
-    sparams['maxiter'] = 12
+    sparams['maxiter'] = 16
```

Comment:
Try 1e-6 or 1e-7... for 16 processors, this might already give you quite a lot of iterations. It seems not to change too much over the simulation, but I did not take a systematic look at it.